nestjs/src/gratitudes/gratitudes.service.ts

```diff
@@ -54,7 +52,12 @@ export class GratitudesService {
     });
   }
 
-  private async saveFeedback(feedback: Feedback) {
+  private async postUserFeedback(data: Feedback) {
+    const feedback = await this.createFeedback(data);
+    await this.postToDiscord(feedback);
+  }
```

Comment:
I am not sure if the naming is accurate :D
I also though about naming it more verbose, like: `saveFeedbackAndPostToDiscord`

Reply:
`createAndNotify` ?